src/main/java/core/basesyntax/model/Circle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.model;
+
+public class Circle extends Figure {
+    private static final double PI = 3.14;
```

Comment:
Maybe it would be better to use [Math.PI](https://docs.oracle.com/javase/8/docs/api/java/lang/Math.html#PI)?